README.md

```diff
@@ -37,6 +37,49 @@ For static use cases, wRPC provides [WIT] binding generators for:
 
 wRPC fully supports the unreleased native [WIT] `stream` and `future` data types along with all currently released WIT functionality.
 
+## Quickstart
+
+Here is a guide demonstrating how to bootstrap the WASM component based example implemented by
+[examples/rust/hello-component-server](examples/rust/hello-component-server) and 
+[examples/rust/hello-component-client](examples/rust/hello-component-client).
+
+### Requirement
+- docker >= 24.0.6, build ed223bc
+- rust >= 1.80.1
+
+### How-To
+1. Add the `wasm32-wasi` target
+    ```bash
+    rustup target add wasm32-wasi
```

Comment:
We should probably use `wasm32-wasip1`, which is the standard name for legacy WASI version https://blog.rust-lang.org/2024/04/09/updates-to-rusts-wasi-targets.html

I'd prefer `wasm32-wasip2` here, but I think we should stick to `wasm32-wasip1`, which is available in stable

Perhaps we could add a note that `wasm32-wasip2` can also be used?